ansible/roles/dom0-defaults/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: install some common packages
+  apt:
+    name: "{{ item }}"
+    state: latest
+    update_cache: yes
+    cache_valid_time: 28800
+    install_recommends: false
+  with_items:
+    - ca-certificates
+    - tmux
```

Comment:
> I'm not against that (and nothing can prevent you from installing it :-P), but I would like to know what use-case is covered better with screen.

Nothing particular, it's just a matter of habit. I know how to use screen fairly well, but know how to use tmux far less.